manager/assets/modext/core/modx.component.js

```diff
@@ -7,8 +7,9 @@ MODx.Component = function(config) {
     if (this.config.tabs) {
         this._loadTabs();
     }
-    this._loadComponents();
     this._loadActionButtons();
+	console.log(document.getElementById('modx-container').innerHTML);
```

Comment:
Lol whoops! That's what I get for submitting a pr during a MODXpo lecture. I'll fix it. 



—
Sent from Mailbox

On Sat, Nov 21, 2015 at 11:20 PM, Mike Reid <notifications@github.com>
wrote:

>>      this._loadActionButtons();
>> +	console.log(document.getElementById('modx-container').innerHTML);
> Any reason this shouldn't be removed?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/modxcms/revolution/pull/12776/files#r45551647